### PR TITLE
Remove deprecated plugins in full edition

### PR DESCRIPTION
**Pull request by Leilei332**
Remove plugins that are marked deprecated in full edition.

### CODE REVIEW

File: editions/full/tiddlywiki.info [hunk 1]
```diff
@@ -2,23 +2,18 @@
...
 		"tiddlywiki/bibtex",
-		"tiddlywiki/blog",
 		"tiddlywiki/browser-sniff",
 		"tiddlywiki/browser-storage",
-		"tiddlywiki/cecily",
 		"tiddlywiki/classictools",
 		"tiddlywiki/codemirror",
 		"tiddlywiki/comments",
-		"tiddlywiki/d3",
-		"tiddlywiki/github-fork-ribbon",
```
Comment (pmario) [inline editions/full/tiddlywiki.info:13]: I did update the github fork ribbon. It should work as expected again.

Comment (Leilei332) [reply editions/full/tiddlywiki.info:13]: I wonder whether marking the github fork ribbon plugin as "deprecated" is done by mistake. IMO it should have been marked "Legacy" or "Stable".

Comment (Jermolene) [reply editions/full/tiddlywiki.info:13]: Yes I think that's right, it should be marked "stable"

**Comment by Jermolene** [comment]
Thanks @Leilei332

**Pull request merged**